src/abi/TokenPool.ts

```diff
@@ -0,0 +1,1078 @@
+export default [
```

Comment:
Should we add v number to name for disambiguation? eg `TokenPool_v1_5.ts`. If we need to add compat for v1.4 of applyChainUpdates() it may be clearer by adding v number in the name. WDYT?